README.md

```diff
@@ -64,8 +64,10 @@ Option | Description
 ---    | ---
 `--latest` | Get the `latest` versions of all components, referring to their `develop` branches.
 `--no-pleuston` | Start up Ocean without an instance of `pleuston`. Helpful for development on `pleuston`.
-`--local-parity-node` | Runs a local parity POA node and Secret Store instead of ganache-cli.
+`--local-secret-store` | Runs a local parity POA node and Secret Store instead of ganache-cli.
 `--reuse-database` | Start up Ocean and reuse the Database from ganache. Helpful for development.
+`--testnet-parity-node` | Start up a parity client connected to Ocean testnet.
```

Comment:
does this really connect to the testnet in aws? or is ist setting up an own testnet?